### ISSUE TITLE
您好，训练中打出了一些信息

### ISSUE THREAD

**Issue by zhangyunming** [yjh0410/YOWOv2#4]
[INFO] Register count_convNd() for <class 'torch.nn.modules.conv.Conv2d'>.
[INFO] Register count_bn() for <class 'torch.nn.modules.batchnorm.BatchNorm2d'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.activation.ReLU'>.
[WARN] Cannot find rule for <class 'torch.nn.modules.container.Sequential'>. Treat it as zero Macs and zero Params.
[INFO] Register zero_ops() for <class 'torch.nn.modules.pooling.MaxPool2d'>.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_backbone.ShuffleV2Block'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_backbone.ShuffleNetV2'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_basic.Conv'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_neck.SPPF'>. Treat it as zero Macs and zero Params.
[INFO] Register count_relu() for <class 'torch.nn.modules.activation.LeakyReLU'>.
[WARN] Cannot find rule for <class 'torch.nn.modules.container.ModuleList'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_fpn.ELANBlock'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_fpn.PaFPNELAN'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free_head.DecoupledHead'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.cnn_2d.yolo_free.yolo_free.FreeYOLO'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_2d.backbone_2d.Backbone2D'>. Treat it as zero Macs and zero Params.
[INFO] Register count_convNd() for <class 'torch.nn.modules.conv.Conv3d'>.
[INFO] Register count_bn() for <class 'torch.nn.modules.batchnorm.BatchNorm3d'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.pooling.MaxPool3d'>.
[WARN] Cannot find rule for <class 'models.backbone.backbone_3d.cnn_3d.shufflnetv2.InvertedResidual'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_3d.cnn_3d.shufflnetv2.ShuffleNetV2'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.backbone.backbone_3d.backbone_3d.Backbone3D'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.basic.conv.Conv2d'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'torch.nn.modules.activation.Softmax'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.yowo.encoder.CSAM'>. Treat it as zero Macs and zero Params.
[INFO] Register zero_ops() for <class 'torch.nn.modules.dropout.Dropout'>.
[WARN] Cannot find rule for <class 'models.yowo.encoder.ChannelEncoder'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.yowo.head.DecoupledHead'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class 'models.yowo.yowo.YOWO'>. Treat it as zero Macs and zero Params.
/home/ubuntu/anaconda3/lib/python3.8/site-packages/torch/functional.py:568: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  ../aten/src/ATen/native/TensorShape.cpp:2228.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
==============================
FLOPs : 1.25 G
Params : 3.52 M
==============================
Optimizer: adamw
--momentum: 0.9
--weight_decay: 0.0005
==============================
WarmUpScheduler: linear
--base_lr: 0.0001
--warmup_factor: 0.00066667
--wp_iter: 500
tensor([[0.2557, 0.0974, 0.5837, 0.8539]], device='cuda:0')
tensor([[0.6044, 0.1925, 0.8323, 0.6527]], device='cuda:0')
tensor([[0.3208, 0.2667, 0.5094, 0.9091]], device='cuda:0')
tensor([[0.3756, 0.0000, 0.7563, 0.8317]], device='cuda:0')
tensor([[0.3898, 0.0000, 0.6712, 0.4589]], device='cuda:0')
tensor([[0.4688, 0.2174, 0.6761, 0.7488]], device='cuda:0')
tensor([[0.0010, 0.3571, 0.3080, 0.9990]], device='cuda:0')
tensor([[0.2393, 0.0474, 0.5123, 0.8147]], device='cuda:0')
[Epoch: 1/8][Iter: 0/42229][lr: 0.000000][loss_conf: 15.85][loss_cls: 1.40][loss_box: 0.87][losses: 21.58][time: 1.72]
tensor([[0.3943, 0.2675, 0.6057, 0.9424],
        [0.1434, 0.2716, 0.3835, 0.9300]], device='cuda:0')
tensor([[0.4412, 0.0667, 0.6581, 0.6292],
        [0.4559, 0.0750, 0.5735, 0.3875]], device='cuda:0')
tensor([[0.5899, 0.0000, 0.7022, 0.2686],
        [0.3399, 0.0289, 0.5028, 0.2934]], device='cuda:0')
tensor([[0.6540, 0.3468, 0.8635, 0.9009]], device='cuda:0')
tensor([[0.3647, 0.0000, 0.7098, 0.6789]], device='cuda:0')
tensor([[0.5238, 0.0167, 0.8095, 0.6542]], device='cuda:0')
tensor([[0.5495, 0.0000, 0.8435, 0.7583],
        [0.3706, 0.2844, 0.8211, 0.7204]], device='cuda:0')
tensor([[0.1012, 0.0159, 0.3742, 0.7729]], device='cuda:0')
tensor([[0.2273, 0.2603, 0.3977, 0.7111],
        [0.0227, 0.2857, 0.3523, 0.7175]], device='cuda:0')

这些tensor怎么自动打出，看起来不太正常，我用的1.11torch

**Comment by yjh0410** [comment]
@zhangyunming 调试时留下的无用代码，已删除。

**Comment by zhangyunming** [comment]
好的，已经注释，ok了

**Comment by snehashis1997** [comment]
how to add blank background images to reduce false positives?
